expression_data_matrix.md

```diff
@@ -40,6 +40,36 @@ The bundle metadata is intended to contain metadata common to all the samples an
 
 It may occur that the matrix contains samples from multiple projects or studies.  If elements like the processing pipeline are no longer common to all samples, care should be taken to move this metadata to the samples or features metadata table instead of the bundle metadata.
 
+# Feature Metadata Discussion
+
+### Minimum Feature Metadata
+
+The following metadata fields describe a recommended minimal set of feature metadata.
+
+<table>
+<tr markdown="block"><td>
+`featureName`
+</td><td>
+_string_
+</td><td>
+A human readable name for the feature. Would be better if it were unique to the sample within the database.
+</td></tr>
+<tr markdown="block"><td>
+`featureType`
+</td><td>
+_string_
+</td><td>
+The annotated type of the feature.  Examples include: Gene, Transcript, Exon, Intron and so on
+</td></tr>
+<tr markdown="block"><td>
+`chromosome`
```

Comment:
Should we also add `start` and `end` position of the feature? Maybe we could use a single field `location` that encodes this.